src/js/common/timezone.js

```diff
@@ -67,8 +67,18 @@ function getCustomTimezoneOffset(timestamp) {
 function getLocalTime(time) {
     var timezoneOffset = getTimezoneOffset(time);
     var customTimezoneOffset = getCustomTimezoneOffset(time);
-    var timezoneOffsetDiff = customTimezoneOffset ? 0 : nativeOffsetMs - timezoneOffset;
-    var localTime = time - customTimezoneOffset + timezoneOffset + timezoneOffsetDiff;
+    var localTime = time - customTimezoneOffset + timezoneOffset;
+    var newDateTimezoneOffsetMS = new Date(localTime).getTimezoneOffset() * MIN_TO_MS;
```

Comment:
The case of `"I'm in Korea not using DST and customTimezone is using DST like New York"` should be handled.

Reply:
I consulted with you.
I decided to use `Intl.DateTimeFormat`.